lib/services/ios-project-service.ts

```diff
@@ -379,5 +393,57 @@ class IOSProjectService extends projectServiceBaseLib.PlatformProjectServiceBase
 			this.$fs.rename(path.join(fileRootLocation, oldFileName), path.join(fileRootLocation, newFileName)).wait();
 		}).future<void>()();
 	}
+	
+	private prepareDynamicFrameworks(pluginPlatformsFolderPath: string, pluginData: IPluginData): IFuture<void> {
+		return (() => {
+			_.each(this.getAllDynamicFrameworksForPlugin(pluginData).wait(), fileName => this.addLibrary(path.join(pluginPlatformsFolderPath, fileName)).wait());			
+		}).future<void>()();
+	}
+	
+	private prepareCocoapods(pluginPlatformsFolderPath: string): IFuture<void> {
+		return (() => {
+			let pluginPodFilePath = path.join(pluginPlatformsFolderPath, "Podfile");
+			if(this.$fs.exists(pluginPodFilePath).wait()) {
+				let pluginPodFileContent = this.$fs.readText(pluginPodFilePath).wait();
+				let contentToWrite = this.buildPodfileContent(pluginPodFilePath, pluginPodFileContent);
+				this.$fs.appendFile(this.projectPodFilePath, contentToWrite).wait();
```

Comment:
If we have two dependencies with incompatible versions `pod install` command fails with descriptive message.

If Podfile has more than one `post_install` sections only the last one is executed.